test/fixtures/errors/unhandled_promise_trace_warnings.snapshot

```diff
@@ -9,6 +9,7 @@
     at *
     at *
     at *
+    at *
```

Comment:
Should we use `Error.captureStackTrace` to avoid this extra frame?

Reply:
![image](https://github.com/nodejs/node/assets/5059100/9593da56-4673-4a57-bc01-7d58a5c504ad)

alwaysWarnUnhandledRejectionsMode is now also in the stack trace. I think it is neglectable.